src/hid-asus.c

```diff
@@ -0,0 +1,320 @@
+/*
+ *  HID driver for Asus notebook built-in keyboard.
```

Comment:
It looks like the touchpad below is an Elan one. So maybe we should just name it hid-elan after all.